src/mlpack/tests/gmm_test.cpp

```diff
@@ -28,7 +28,7 @@ using namespace mlpack::gmm;
 
 BOOST_AUTO_TEST_SUITE(GMMTest);
 /**
- * Test GMM::Probability() for a single observation for a few cases.
+ * Test GMM::Probability() for a different observation for a few cases.
```

Comment:
Ah, I think the original intent of this comment was to say we are calling `gmm.Probability()` with only one observation at a time (e.g., we are doing `gmm.Probability("0 0")` instead of `gmm.Probability("0 0; 1 1")`).  Maybe `single` was not the best word to describe that, possibly `Test GMM::Probability() with a single observation at a time for a few cases`?  Up to you, maybe you have a better idea. :)

Reply:
Resolved :)